src/plays/download-fetch-response/DownloadFetchResponse.jsx

```diff
@@ -0,0 +1,113 @@
+import {
+  Alert,
+  Box,
+  Button,
+  Card,
+  CardActions,
+  CardContent,
+  Grid,
+  Snackbar,
+  Typography,
+} from "@mui/material";
+import { useEffect, useState } from "react";
+import { UseCustomFetchAPI } from "./CustomHooks/UseCustomFetchAPI";
+import "./response.scss";
+
+export default function DownloadFetchResponse() {
+  const [endpoint, setEndpoint] = useState("");
+  const [alertError, setAlertError] = useState();
+  const [toBeFetched, setToBeFetched] = useState("");
+  const { data, error, fetching } = UseCustomFetchAPI(toBeFetched);
+
+  useEffect(() => {
+    const download = document.createElement("a");
+    download.href = URL.createObjectURL(
+      new Blob([JSON.stringify(data, null, 2)], {
+        type: "text/plain",
+      })
+    );
+    download.setAttribute("download", `Play - ${new Date().getTime()}.json`);
+    document.body.appendChild(download);
+    if (!["", undefined, null].includes(toBeFetched?.trim()) && !error)
+      download.click();
+    document.body.removeChild(download);
+  }, [data]);
+
+  useEffect(() => {
+    setAlertError(error);
+  }, [error]);
+
+  const reset = () => {
+    setEndpoint("");
+    setToBeFetched("");
+  };
+  return (
+    <div className="fetch-download">
```

Comment:
@abhi-works 

It seems you have removed all the code that the create-play process generates. That results into losing the play headre. You can cross check with other plays. Can you please bring it back?

Reply:
Got It but the template is not generated with cli npx command

Reply:
That's a problem that we have to debug then and fix in ReactPlay...

In the mean time, you can understand the structure from other plays and do for yours.

Here is an example: https://github.com/reactplay/react-play/blob/main/src/plays/image-gallery/Imagegallery.js

Also please let us know on Discord, what error you get while creating the play. We want to fix that asap.

Reply:
Yes, considering other plays, I have included the header for the time being. Changes are pushed . I'll try to make another play and see if the problem persists @atapas